.github/workflows/update-version.yml

```diff
@@ -0,0 +1,26 @@
+name: update-pack-version
+
+on:
+  repository_dispatch:
+    types:
+      - pack-release
```

Comment:
does this mean the action will be triggered when there's a new release on the pack repo?

Reply:
Yup, wrote a similar workflow for [pack-orb](https://github.com/buildpacks/pack-orb/pull/34)